strongbox-cron/strongbox-cron-tasks/src/main/java/org/carlspring/strongbox/cron/jobs/RegenerateChecksumCronJob.java

```diff
@@ -62,7 +62,7 @@ public void executeTask(CronTaskConfigurationDto config)
 
         if (storageId == null)
         {
-            Map<String, Storage> storages = getStorages();
+            Map<String, StorageData> storages = getStorages();
```

Comment:
Okay.... Similarly to the above comment... I'm not really convinced that we need to rename the `Storage` and `Repository`. I think we should have these two classes with these exact names. (Perhaps, I'm missing the point, as I still haven't fully gone over this massive pull request).

Reply:
@carlspring please share your thoughts about [this](https://github.com/strongbox/strongbox/pull/1359#discussion_r299744909) comment